gen3-integration-tests/gen3_ci/jenkins_jobs/ci-only-fence-create-client.groovy

```diff
@@ -90,17 +90,10 @@ pipeline {
 
                             echo "Running: ${FENCE_CMD}"
                             # execute the above fence command
-                            FENCE_CMD_RES=$(bash -c "${FENCE_CMD}" | tee >(tail -n 1 > temp_client_cred.txt))
-                            file_content=$(cat temp_client_cred.txt)
+                            # execute the above fence command
+                            FENCE_CMD_RES=$(bash -c "${FENCE_CMD}")
 
-                            case "$CLIENT_TYPE" in
-                                "implicit")
-                                    CLIENT_CREDS=$(echo "$file_content" | sed -e "s/(\\('\\(.*\\)'\\),None)/\\2,None/" -e "s/(\\('\\(.*\\)'\\), \\(.*\\))/$CLIENT_NAME: \\2,\\3/")
-                                    ;;
-                                *)
-                                    CLIENT_CREDS=$(echo "$file_content" | sed -e "s/(\\('\\(.*\\)'\\), '\\(.*\\)')/$CLIENT_NAME: \\2,\\3/")
-                            esac
-                            echo $CLIENT_CREDS >> clients_creds.txt
+                            echo "CLIENT_NAME: ${CLIENT_NAME} ${FENCE_CMD_RES}" >> clients_creds.txt
```

Comment:
we should get the last line and save just ${CLIENT_NAME}:${FENCE_CMD_RES} ... then we can split by `:` and literal_eval into a tuple